docs/content/design/segments.md

```diff
@@ -52,4 +165,10 @@ Sharding Data to Create Segments
 
 ### Sharding Data by Dimension
```

Comment:
Is it preferred to call this 'sharding' or 'partitioning'?  I think technically sharding is a particular type of partitioning (horizontal, shared nothing partitioning), so either could work here, it's just a matter of how specific we want to be.  I think it's confusing that the documentation uses both terms---I wondered whether there these referred to different things.